client/src/components/generic/SignUpNotif/SignUpNotif.tsx

```diff
@@ -21,21 +22,30 @@ export const SignUpNotif = ({ signedIn }: ISignUpNotif) => {
   }
 
   return (
-    <div
-      className="text-dark-blue-100 border-gray-3 flex flex-col items-center justify-center 
-    gap-4 rounded-md border bg-white py-8"
-    >
-      <h2 className="text-center italic">Currently unavailable</h2>
-      <h4 className="max-h-[87px] max-w-[386px] text-center">
-        Only paid members can access bookings. Please sign up or wait until your
-        membership payment has been proccessed.
-      </h4>
-      <span className="flex w-full max-w-80 gap-2">
-        {!signedIn && <Button onClick={() => goToLogin()}>Log in</Button>}
-        <Button onClick={() => goToRegister()}>
-          {signedIn ? "Join Now" : "Sign up"}
-        </Button>
-      </span>
+    <div className="bg-gray-1 absolute inset-0 flex h-screen w-full justify-center overflow-hidden bg-opacity-70 px-4 py-2">
```

Comment:
can we not have this absolute, i.e make it the same style as the old one, just with the additional lock?